client/question/components/CodeSnippetDirective.js

```diff
@@ -50,31 +50,30 @@ tie.directive('codeSnippet', [function() {
          */
         $scope.isModalOpen = MonospaceDisplayModalService.isDisplayed;
 
-        // The maximum number of lines to show in the abbreviated code snippet.
-        $scope.MAX_NUM_LINES_IN_ABBREVIATED_SNIPPET = 3;
+        // The maximum number of lines to show in a code snippet.
```

Comment:
Is context lost when we drop "abbreviated"? This is in reference to user code that is > 3 lines in the feedback window and is therefore truncated, right? Would this cause confusion with code snippets that are not truncated (e.g., short code or code displayed in a detailed code pop-up/modal)?

Reply:
I see. I agree that the two terms sound redundant. As long as the convention of calling truncated code "snippet" is consistent throughout, SGTM.